Frameworks/CoreText/DWriteWrapper.mm

```diff
@@ -52,6 +53,17 @@ void __InitDWriteFontPropertiesFromName(
     return s_dwriteFactory;
 }
 
+// Private helper for converting a DWRITE_FONT_METRIC unit to a CTFont API unit
+// DWRITE_FONT_METRICS measures its metrics in 'design units'
+// CTFont APIs generally return in terms of point size
+CGFloat _CoreTextScaleMetric(CGFloat metric, CGFloat pointSizeUnitsPerEm) {
```

Comment:
@ms-jihua suggested a naming convention of double underscore for private to the file, single underscore for private to the project